README.md

```diff
@@ -1,5 +1,7 @@
 # GB/T 2260
 
+[![Build Status](https://travis-ci.org/lichunqiang/GB2260.php.svg)](https://travis-ci.org/lichunqiang/GB2260.php)
```

Comment:
The url should be `cn/GB2260.php`

@lepture @shen2 Please enable `GB2260.php` project at https://travis-ci.org/profile/cn, I don't have permission to to that.